resources/views/ItemList/Components/Filter/ItemFilter.twig

```diff
@@ -1,68 +1,76 @@
         </div>
         <div v-else-if="facet.type === 'image'">
             <div v-if="!facet.noAvailableFiltersText">
                 <div class="form-check" v-for="value in facet.values" :key="value.id">
-                    <input :id="'option-' + value.id" class="form-check-input hidden-xs-up" type="checkbox" :checked="value.selected" @change="updateFacet(value)" :disabled="isLoading">
+                    <input :id="'option-' + value.id" class="form-check-input hidden-xs-up" type="checkbox" :checked="value.selected" @change="updateFacet(value)" :disabled="isLoading" style="display:none">
                     <label :for="'option-' + value.id" class="form-check-label">
                         <img v-show="value.imageUrl" :src="value.imageUrl" width="80px"/>
-                        <span v-show="!value.imageUrl">${ value.name }</span>
+                        <span v-show="!value.imageUrl" v-text="value.name"></span>
                     </label>
-                    <div class="filter-badge">${ value.count }</div>
+                    <div class="filter-badge" v-text="value.count"></div>
                 </div>
             </div>
-            <p v-if="facet.noAvailableFiltersText">${ facet.noAvailableFiltersText }</p>
+            <p v-if="facet.noAvailableFiltersText" v-text="facet.noAvailableFiltersText"></p>
         </div>
         <div v-else-if="facet.type === 'color'">
             <div v-if="!facet.noAvailableFiltersText">
-                <item-color-tiles :facet="facet"></item-color-tiles>
+                <item-color-tiles template-override="#vue-findologic-item-color-tiles" :facet="facet"></item-color-tiles>
             </div>
-            <p v-if="facet.noAvailableFiltersText">${ facet.noAvailableFiltersText }</p>
+            <p v-if="facet.noAvailableFiltersText" v-text="facet.noAvailableFiltersText"></p>
         </div>
         <div v-else-if="facet.id === 'cat'">
             <div v-if="!facet.noAvailableFiltersText" class="form-check" v-for="value in facet.values" :key="value.id">
                 <div class="category-container">
-                    <input :id="'option-' + value.id" class="form-check-input hidden-xs-up" type="checkbox" :checked="value.selected" @change="updateFacet(value)" :disabled="isLoading">
-                    <label :for="'option-' + value.id" class="form-check-label">
-                        ${ value.name }
-                    </label>
-                    <div class="filter-badge">${ value.count }</div>
+                    <input :id="'option-' + value.id"
+                           class="form-check-input hidden-xs-up"
+                           type="checkbox"
+                           :checked="value.selected"
+                           @change="updateFacet(value)"
+                           :disabled="isLoading"
+                           style="display:none"
```

Comment:
I really do not like inline styles like that. Plentymarkets has a separate class for hidden things which is simply `hidden`. Set this class everywhere where it's necessary.